src/api/customer/customer-serializer.ts

```diff
@@ -19,6 +19,74 @@ const createCustomerSerializer: CustomerCreationRequestHandler = (req, res, next
     next();
 }
 
+const paginationSerializer: GetAllCustomerRequestHandler = (req, res, next) => {
+    const queryParams = req.query;
+    let offset = 0;
+    let limit = 50;
+
+    if (queryParams.offset) {
+        offset = Number(queryParams.offset);
+    }
+
+    if (queryParams.limit) {
+        limit = Number(queryParams.limit);
+    }
+
+    res.locals.paginationParamsSerializer = { offset, limit };
+
+    next();
+};
+
+const getAllCustomersSerializer: GetAllCustomerRequestHandler = (req, res, next) => {
+    const allCustomers = res.locals.getCustomer;
+    res.locals.customersToRespond = [];
+
+    allCustomers.map((customer) => {
+        res.locals.customersToRespond.push({
+            uuid: customer.uuid,
+            name: customer.name,
+            contact: {
+                email: customer.email,
+                phone: customer.phone,
+            },
+            document: {
+                cpf: customer.cpf,
+                cnpj: customer.cnpj,
+            },
+            createdAt: customer.createdAt.toISOString(),
+            updatedAt: customer.updatedAt.toISOString(),
+        });
+    });
```

Comment:
Poderia ter feito a atribuição diretamente, ao invés de criar um array vazio para popular depois.

```suggestion
     res.locals.customersToRespond = allCustomers.map((customer) => ({
        uuid: customer.uuid,
        name: customer.name,
        contact: {
            email: customer.email,
            phone: customer.phone,
        },
        document: {
            cpf: customer.cpf,
            cnpj: customer.cnpj,
        },
        createdAt: customer.createdAt.toISOString(),
        updatedAt: customer.updatedAt.toISOString(),
    }));
```